bot/exts/events/advent_of_code/_cog.py

```diff
@@ -52,13 +60,80 @@ def __init__(self, bot: Bot):
         self.status_task.set_name("AoC Status Countdown")
         self.status_task.add_done_callback(_helpers.background_task_callback)
 
+        self.completer_task.start()
```

Comment:
Cancel this when the cog unloads.

Reply:
Done in https://github.com/python-discord/sir-lancebot/pull/991/commits/2fe89228fd3e583f2c8746d7df08af9c1c234972